config/clusters/2i2c/paleohack2021.values.yaml

```diff
@@ -39,9 +34,7 @@ jupyterhub:
       limit: 2
     image:
       name: quay.io/2i2c/paleohack-2021
-      tag: 9d557294938e
-    nodeSelector:
-      2i2c.org/community: paleo
+      tag: 7534858b1098
```

Comment:
Does their user image need to change back too?

Reply:
I think you are right and this is probably a bug caused by just hitting the "revert PR" button